Build issues

Hi,

Build fails from Docker-Compose:

Step 4/19 : RUN mvn install     --quiet         -D license.skip=true    -D maven.test.skip=true
 ---> Running in 159b2190ee29
[ERROR] Unable to find commits until some tag
java.lang.RuntimeException: Unable to find commits until some tag
    at pl.project13.jgit.JGitCommon.findCommitsUntilSomeTag (JGitCommon.java:268)
    at pl.project13.jgit.DescribeCommand.call (DescribeCommand.java:277)
    at pl.project13.maven.git.JGitProvider.getGitDescribe (JGitProvider.java:221)
    at pl.project13.maven.git.JGitProvider.getGitDescribe (JGitProvider.java:117)
    at pl.project13.maven.git.GitDataProvider.maybePutGitDescribe (GitDataProvider.java:167)
    at pl.project13.maven.git.GitDataProvider.loadGitData (GitDataProvider.java:120)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitDataWithJGit (GitCommitIdMojo.java:588)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitData (GitCommitIdMojo.java:555)
    at pl.project13.maven.git.GitCommitIdMojo.execute (GitCommitIdMojo.java:376)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.eclipse.jgit.errors.RevWalkException: Walk failure.
    at org.eclipse.jgit.revwalk.RevWalk.iterator (RevWalk.java:1325)
    at pl.project13.jgit.JGitCommon.findCommitsUntilSomeTag (JGitCommon.java:256)
    at pl.project13.jgit.DescribeCommand.call (DescribeCommand.java:277)
    at pl.project13.maven.git.JGitProvider.getGitDescribe (JGitProvider.java:221)
    at pl.project13.maven.git.JGitProvider.getGitDescribe (JGitProvider.java:117)
    at pl.project13.maven.git.GitDataProvider.maybePutGitDescribe (GitDataProvider.java:167)
    at pl.project13.maven.git.GitDataProvider.loadGitData (GitDataProvider.java:120)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitDataWithJGit (GitCommitIdMojo.java:588)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitData (GitCommitIdMojo.java:555)
    at pl.project13.maven.git.GitCommitIdMojo.execute (GitCommitIdMojo.java:376)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.eclipse.jgit.errors.MissingObjectException: Missing commit 2a254845fa4c0192908f6dc770a7cd23fc9fbe66
    at org.eclipse.jgit.internal.storage.file.WindowCursor.open (WindowCursor.java:147)
    at org.eclipse.jgit.revwalk.RevWalk.getCachedBytes (RevWalk.java:903)
    at org.eclipse.jgit.revwalk.RevCommit.parseHeaders (RevCommit.java:150)
    at org.eclipse.jgit.revwalk.PendingGenerator.next (PendingGenerator.java:147)
    at org.eclipse.jgit.revwalk.StartGenerator.next (StartGenerator.java:184)
    at org.eclipse.jgit.revwalk.RevWalk.next (RevWalk.java:435)
    at org.eclipse.jgit.revwalk.RevWalk.iterator (RevWalk.java:1323)
    at pl.project13.jgit.JGitCommon.findCommitsUntilSomeTag (JGitCommon.java:256)
    at pl.project13.jgit.DescribeCommand.call (DescribeCommand.java:277)
    at pl.project13.maven.git.JGitProvider.getGitDescribe (JGitProvider.java:221)
    at pl.project13.maven.git.JGitProvider.getGitDescribe (JGitProvider.java:117)
    at pl.project13.maven.git.GitDataProvider.maybePutGitDescribe (GitDataProvider.java:167)
    at pl.project13.maven.git.GitDataProvider.loadGitData (GitDataProvider.java:120)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitDataWithJGit (GitCommitIdMojo.java:588)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitData (GitCommitIdMojo.java:555)
    at pl.project13.maven.git.GitCommitIdMojo.execute (GitCommitIdMojo.java:376)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)

--- a/Dockerfile
+++ b/Dockerfile
@@ -21,7 +21,7 @@ RUN wget \
 	--directory-prefix /usr/local/tomcat/lib \
 	https://jdbc.postgresql.org/download/postgresql-42.2.6.jar
 
-COPY --from=build /app/onebusaway-transit-data-federation-builder/target/onebusaway-transit-data-federation-builder-2.0.0-SNAPSHOT-withAllDependencies.jar /app
+COPY --from=build /app/onebusaway-transit-data-federation-builder/target/onebusaway-transit-data-federation-builder-2.0.1-SNAPSHOT-withAllDependencies.jar /app
 
 COPY --from=build /app/onebusaway-transit-data-federation-webapp/target/onebusaway-transit-data-federation-webapp.war /tmp
 RUN unzip \

--- a/bin/ci
+++ b/bin/ci
@@ -41,7 +41,7 @@ echo "=== building bundle"
 if ! docker run --rm -v "$(pwd)/ci-bundle:/bundle" oba-ci bash -c 'cd /bundle && \
        curl --max-time 10 --retry 3 --fail -L -o /tmp/data.zip http://gtfs.halifax.ca/static/google_transit.zip && \
        java -Xss4m -Xmx3G \
-            -jar /app/onebusaway-transit-data-federation-builder-2.0.0-SNAPSHOT-withAllDependencies.jar \
+            -jar /app/onebusaway-transit-data-federation-builder-2.0.1-SNAPSHOT-withAllDependencies.jar \
             /tmp/data.zip .' >/tmp/build-bundle.out 2>&1; then
     echo " ! bundle building failed"
     echo "=== bundle build output"

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -13,7 +13,7 @@ services:
     # Also, the JAR must be executed from within the same
     # directory as the bundle, or else some necessary files
     # are not generated
-    command: ["sh", "-c", "cd /bundle && java -Xss4m -Xmx3G -jar /app/onebusaway-transit-data-federation-builder-2.0.0-SNAPSHOT-withAllDependencies.jar $$(ls /bundle/*.zip) ."]
+    command: ["sh", "-c", "cd /bundle && java -Xss4m -Xmx3G -jar /app/onebusaway-transit-data-federation-builder-2.0.1-SNAPSHOT-withAllDependencies.jar $$(ls /bundle/*.zip) ."]
 
   # The `oba` service expects this service to be addressed
   # as `postgres`, so this name cannot be changed

